models/repo.go

```diff
@@ -1231,7 +1231,7 @@ func CountUserRepositories(userID int64, private bool) int64 {
 }
 
 // Repositories returns all repositories
-func Repositories(opts *SearchRepoOptions) (_ RepositoryList, err error) {
+func Repositories(opts *SearchRepoOptions) (_ RepositoryList, count int64, err error) {
```

Comment:
Since `Repositories(..)` only looks at `opts.PageSize`, `opts.Page`, and `opts.OrderBy`, would it be better to have it take 3 arguments?